CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ and **Merged pull requests**. Critical items to know are:
 Referenced versions in headers are tagged on Github, in parentheses are for pypi.
 
 ## [vxx](https://github.com/pydicom/deid/tree/master) (master)
- - bug fix for multivalued fields in %values lists [#174](https://github.com/pydicom/deid/issues/174) (0.2.25)
+ - updated pydicom dependency from 1.3.0 to 2.1.1 [#171] (https://github.com/pydicom/deid/issues/171), bug fix for multivalued fields in %values lists [#174](https://github.com/pydicom/deid/issues/174) (0.2.25)
```

Comment:
What I usually do for larger releases is have multiple lines with comments, and just put the release number at the top. You could also add an indentation if you like (but not necessary!) It just is easier for readablity. 
```suggestion
 - updated pydicom dependency from 1.3.0 to 2.1.1 [#171] (https://github.com/pydicom/deid/issues/171) (0.2.25)
 - bug fix for multivalued fields in %values lists [#174](https://github.com/pydicom/deid/issues/174)
```

Reply:
That's much better! Thanks!